doc/concepts/cable_cell.rst

```diff
@@ -9,15 +9,39 @@ channels, synapses, gap junction sites, stimuli and spike detectors.
 
 Cable cells are constructed from three components:
 
-* :ref:`Morphology <morph>`: a decription of the geometry and branching structure of the cell shape.
-* :ref:`Label dictionary <labels>`: a set of rules that refer to regions and locations on the cell.
+* :ref:`Morphology <morph>`: a description of the geometry and branching structure of the cell shape.
+* :ref:`Label dictionary <labels>`: a set of definitions and a :abbr:`DSL (domain specific language)` that refer to regions and locations on the cell morphology.
 * :ref:`Decor <cablecell-decoration>`: a description of the dynamics on the cell, placed according to the named rules in the dictionary.
 
-When a cable cell is constructued the following steps are performed using the inputs:
+The decor can make reference to particular mechanisms and mechanism catalogues. More on mechanisms at :ref:`mechanisms`.
 
-1. Concrete regions and locsets are generated for the morphology for each labeled region and locset in the dictionary
+
+.. graphviz::
+  :align: center
+  :caption: The three main ingredients of the cable cell: morphology, labels, decor. A defined morphology enables referring to parts of it through the label DSL, references which can be stored as named labels. A decor places (or paints) properties such as mechanisms onto precise places (labels) on the morphology. Straight lines represent inputs, dashed lines references.
```

Comment:
The sentence is a bit unclear. Maybe something like this? 
`The three main ingredients of the cable cell: morphology, label dictionary, decor. Regions and locations of the morphology can be identified using a DSL (maybe link to the DSL?). The definitions can be stored with a label in the label dictionary. The regions and locsets can then be referred to using those labels when painting and placing dynamics and properties using a decor. Straight lines represent inputs, dashed lines references.`